docs/source/en/guides/download.md

```diff
@@ -187,16 +187,20 @@ For more details about the CLI download command, please refer to the [CLI guide]
 
 ## Faster downloads
 
-If you are running on a machine with high bandwidth, you can increase your download speed with [`hf_transfer`](https://github.com/huggingface/hf_transfer), a Rust-based library developed to speed up file transfers with the Hub. To enable it, install the package (`pip install hf_transfer`) and set `HF_HUB_ENABLE_HF_TRANSFER=1` as an environment variable.
+If you are running on a machine with high bandwidth,
+you can increase your download speed with [`hf_transfer`](https://github.com/huggingface/hf_transfer),
+a Rust-based library developed to speed up file transfers with the Hub.
+To enable it:
 
-<Tip>
-
-Progress bars are supported in `hf_transfer` starting from version `0.1.4`. Consider upgrading (`pip install -U hf-transfer`) if you plan to enable faster downloads.
```

Comment:
By pinning `0.1.4` in `setup.py`'s extra, I think it's fine to remove this `Tip`